op-node/rollup/finality/plasma_test.go

```diff
@@ -137,9 +147,16 @@ func TestPlasmaFinalityData(t *testing.T) {
 				L1Origin:       previous.ID(), // reference previous origin, not the block the batch was included in
 				SequenceNumber: j,
 			}
+			if i < 10 {
```

Comment:
can we extract 10 as a const above the for loop? at line 120

```
XXX := 10
for i := uint64(0); i < 200; i++ {
    if i == XXX {
    }
}
```

Reply:
sure, updated.